crud.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+// Inserting the record
+func (apiCfg apiConifg) insertRecord(record any) (interface{}, error) {
+	// Maybe extract the dbName and the collectionName from .env file?
```

Comment:
I believe yes

Reply:
But when we have multiple different collections and/dbs, wouldn't it kindof become annoying to keep track of it all in that one file?